library/core/src/ptr/mod.rs

```diff
@@ -881,12 +881,17 @@ pub unsafe fn read_unaligned<T>(src: *const T) -> T {
 #[inline]
 #[stable(feature = "rust1", since = "1.0.0")]
 pub unsafe fn write<T>(dst: *mut T, src: T) {
-    if cfg!(debug_assertions) && !is_aligned_and_not_null(dst) {
-        // Not panicking to keep codegen impact smaller.
-        abort();
+    extern "rust-intrinsic" {
+        fn copy_nonoverlapping<T>(src: *const T, dst: *mut T, count: usize);
+    }
```

Comment:
Add a comment on why it's linked again instead of using already imported `crate::intrinsics::copy_nonoverlapping` as done by other functions in this module?

Reply:
I will, if the reviewers agree that the PR should land at all.

The reason is to avoid the overhead of another function call indirection.